peer-prep/src/Utility/localStorage.js

```diff
@@ -0,0 +1,23 @@
+import QuestionDatabase from "../DataModel/QuestionDatabase";
+
+function updateLocalQuestionDatabase(updatedDatabase) {
+    const data = JSON.stringify(Array.from(updatedDatabase.database.entries()));
+    localStorage.setItem("questions", data);
+}
+
+function retrieveQuestionDatabase() {
+    let questionDatabase = new QuestionDatabase();
+
+    questionDatabase.setDatabase(new Map(JSON.parse(localStorage.getItem("questions"))))
+    return questionDatabase;
+}
+
+function setupQuestionDatabase() {
+    var data = localStorage.getItem("questions");
+
+    if(!data) {
+        localStorage.setItem("questions", JSON.stringify([]))
+    };
+}
```

Comment:
Great utility functions have been provided for data storage! 💯 Remember to modify them once our real database has been configured.